bin/crew

```diff
@@ -132,14 +137,6 @@ class ExitMessage
 end
 
 at_exit do
-  begin
-    gem 'activesupport'
-  rescue Gem::LoadError
-    puts ' -> install gem activesupport'
-    Gem.install('activesupport')
-    gem 'activesupport'
-  end
-  require 'active_support/core_ext/object/blank'
   GnomePostinstall.run unless GnomePostinstall.gnome_packages.blank?
```

Comment:
can't we just use `.empty?` here

Reply:
The great thing about `blank?` is that it avoids `NoMethodError`s.

See https://stackoverflow.com/a/26877095

Reply:
And yes, I used to think the same thing, and then just got sick of it and accepted that adding `.blank?` from the `activesupport` gem really makes life so much easier.